test/distributed/fsdp/test_fsdp_misc.py

```diff
@@ -857,13 +857,13 @@ def test_no_params(self):
         torch.cuda.set_device(self.rank)
         # Test CPU
         no_params = nn.ReLU()
-        module = FSDP(no_params)
+        module = FSDP(no_params)  # noqa: F841
         # Test CUDA
         no_params = nn.ReLU().cuda()
         module = FSDP(no_params)
```

Comment:
This one would need a noqa as well?

Reply:
I got rid of all of them, there's no reason to believe those variables need to exist.